drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/util/ScenarioBeanUtil.java

```diff
@@ -79,6 +83,11 @@ public static Object navigateToObject(Object rootObject, List<String> steps) {
         for (String step : steps) {
             Field declaredField = null;
             try {
+                if(currentObject == null) {
```

Comment:
- Could we please update formatting to: `if (currentObject == null) {`
- And thinking if it shouldn't be `if (currentClass == null) {` as this if block access `currentClass` and `currentObject` is not used inside.

Reply:
Done reformat but the code is correct because I need to check if `currentObject` is `null` from the previous iteration. It can be `null` only if it is the last step.